packages/graphql-server-reactive-core/src/messageTypes.ts

```diff
@@ -0,0 +1,37 @@
+import { ExecutionResult } from 'graphql';
+
+// Refer to https://github.com/apollographql/graphql-server/issues/272#issuecomment-278805955 for more information about protocol
```

Comment:
I think we need to discuss the transport and message types with Uri and the folks working on GraphQL subscriptions. It makes a whole lot of sense to have a shared transport, so they probably have their opinions about the message types as well.